predict.ecl

```diff
@@ -1,6 +1,8 @@
 ﻿// Produce a data set of predictions for each model
-IMPORT $ as SVM;
-IMPORT $.LibSVM;
+IMPORT SupportVectorMachines as SVM;
```

Comment:
It is usually better to use $ AS SVM if you are using Eclipse or VS Code.  Does the ECL IDE give you a problem if you don't use the directory name?